src/main/java/it/gov/pagopa/pu/migration/wf/service/ingestion/debtposition/DebtPositionProcessingService.java

```diff
@@ -0,0 +1,162 @@
+package it.gov.pagopa.pu.migration.wf.service.ingestion.debtposition;
+
+import com.opencsv.exceptions.CsvException;
+import it.gov.pagopa.pu.migration.model.Uploads;
+import it.gov.pagopa.pu.migration.service.file.CsvService;
+import it.gov.pagopa.pu.migration.wf.dto.debtposition.DebtPositionErrorDTO;
+import it.gov.pagopa.pu.migration.wf.dto.debtposition.DebtPositionMigrationFileResult;
+import it.gov.pagopa.pu.migration.wf.dto.debtposition.InstallmentIngestionFlowFileDTO;
+import it.gov.pagopa.pu.migration.wf.service.ingestion.ErrorArchiverService;
+import it.gov.pagopa.pu.migration.wf.service.ingestion.MigrationProcessingService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Lazy
+@Slf4j
+public class DebtPositionProcessingService extends MigrationProcessingService<InstallmentIngestionFlowFileDTO, DebtPositionMigrationFileResult, DebtPositionErrorDTO> {
```

Comment:
why should you extend `MigrationProcessingService` if you are bypassing its public method?
probably you should not extend it, or you should not expose a new public method, instead just overriding the required method in order to inherit the common behavior